app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -62,6 +62,7 @@ def show
     param :redhat_repository_url, String, :desc => N_("Red Hat CDN URL")
     param_group :resource, ::Api::V2::TaxonomiesController
     def update
+      params.delete(:users) if params[:users]  # Cannot PUT a GET result as the UI does
```

Comment:
I suggested that users be removed from the attr_accessible on the taxonomy but apparently a usage of Organization.new() is required. Trying to update an org w/ a users param fails with the error. The katello UI gets org json, modifies some of it, then PUTs it back. Foreman UI doesn't use API so they don't encounter this.